bigquery_etl/docs/validate_docs.py

```diff
@@ -38,20 +38,21 @@ def validate(project_dirs):
             for root, dirs, files in os.walk(project_dir):
                 if os.path.basename(root) == EXAMPLE_DIR:
                     for file in files:
-                        dry_run_sql = sub_local_routines(
-                            (Path(root) / file).read_text(),
-                            project_dir,
-                            parsed_routines,
-                        )
+                        if os.path.splitext(file)[1] == ".sql":
```

Comment:
Run validation only on SQL files

Reply:
We could avoid this extra level of nesting by moving the extension check into the `for` above:

```
sql_files = (f for f in files if os.path.splitext(f)[1] == ".sql")
for file in sql_files:
```